Gems/Atom/Utils/Code/CMakeLists.txt

```diff
@@ -27,6 +27,27 @@ ly_add_target(
             3rdParty::libpng
 )
 
+if(PAL_TRAIT_BUILD_HOST_TOOLS)
+
+    ly_add_target(
+        NAME Atom_Utils.Editor.Static STATIC
```

Comment:
I'm confused by the "Editor" word. Should shouldn't this project be called "Atom_Utils.TestUtils.Static" or something like that?

Reply:
I chose "Editor" because it is including AzToolsFramework for EditorAssetSystemAPI.h. Is there a distinction between tests that use editor code and those that don't?

Reply:
I don't know if we distinguish between test code for editor vs general test code.
I think "Test" is the more relevant information than "Editor". When I see the "Editor", I would would not expect the library to contain test code. When I see "Test", I would not be surprised to find editor code in it.